docs/eventing/sources/sinkbinding/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "SinkBinding"
+weight: 01
+type: "docs"
+aliases:
+  - /docs/eventing/samples/sinkbinding/index
+  - /docs/eventing/samples/sinkbinding/README
+  - /docs/eventing/sources/sinkbinding.md
+---
+
+![API version v1](https://img.shields.io/badge/API_Version-v1-red?style=flat-square)
+
+The `SinkBinding` custom object supports decoupling event production from
+delivery addressing.
```

Comment:
I'd go with either "object" or "custom resource" instead of _custom object_

Reply:
Changed to "object"